DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -31,6 +32,9 @@
                                  es_dqm_source_offline *
                                  castorSources *
                                  HcalDQMOfflineSequence )
+eras.phase1Pixel.toReplaceWith(DQMOfflinePreDPG, DQMOfflinePreDPG.copyAndExclude([ # FIXME
+    siPixelOfflineDQM_source, # Pixel DQM needs to be updated for phase1
```

Comment:
@makortel Do I uderstand correctly that the new Pixel DQM should/could be plugged in here?

I changed https://github.com/schneiml/cmssw/blob/phase1pixeldqm-histoman-refactoring/DQM/SiPixelCommon/python/SiPixelOfflineDQM_source_cff.py#L133 to introduce eras, which seems to be another alternative to me.